### ISSUE TITLE
Should the Rails Camp code of conduct match the RubyNZ one?

### ISSUE THREAD

**Issue by jimsynz** [rubynz/camp#10]
The RubyNZ CoC has been anguished over at length, and contains the sentence:

> This code of conduct applies to all Ruby New Zealand spaces, including our associated mailing lists, _events_, Slack group, and projects; both online and off. Anyone who violates this code of conduct may be sanctioned or expelled from these spaces at the discretion of the Ruby New Zealand committee.

Given that RubyNZ has a hand in organising and paying for Rails Camp, as well as has it's annual AGM there, I suspect most people would expect that Rails Camp is a "Ruby New Zealand event".

Should we replace the CoC info on this site with information about the RubyNZ CoC?

**Comment by megahbite** [comment]
:+1:

It was the intention for it to cover Rails Camp as well as it's a Ruby NZ event.

**Comment by parndt** [comment]
Link to it instead of what is there now? Maintaining only one is better

**Comment by mermop** [comment]
👍

**Comment by terrcin** [comment]
> Given that RubyNZ has a hand in organising and paying for Rails Camp, as well as has it's annual AGM there, I suspect most people would expect that Rails Camp is a "Ruby New Zealand event".

Just an FYI: RubyNZ isn't at all paying for Rails Camp, it's in fact the other way around as we're paying for your tickets. $25 from each ticket sold is going towards your tickets.

Edit: budget is tight at the moment, so I was looking at where the money was going and the committee tickets stood out sitting at $1800.

Edit2: Having chatted with Phil there was a misunderstanding in how committee tickets were covered and we'd budget for the camp paying for them. Turns out that Ruby NZ is covering them and we've updated our budget accordingly :-)
